packages/form-builder/src/types/formState.ts

```diff
@@ -7,4 +7,5 @@ import type {FormSettings} from './formSettings';
 export type FormState = {
     blocks: Block[];
     settings: FormSettings;
+    isDirty?: boolean;
```

Comment:
We should consider introducing the concept of Editor State at some point in the next cycle. I think it would be better to delineate Form State as persisted state. By contrast, `isDirty` does not persist with the form.

Reply:
There are also scattered lines of `useState` for things like sidebar visibility, etc, that could be included in Editor State.

Reply:
I agree